src/main/java/pl/edu/agh/mwo/reporter/Main.java

```diff
@@ -33,45 +37,96 @@ public static void main(String[] args) throws IOException, ParseException {
             CommandLine cmd = parser.parse(options, args);
             String directory = cmd.getOptionValue("source");
             String rType = cmd.getOptionValue("rtype");
+            LocalDate dateFrom = null;
```

Comment:
Dałoby się podzielić tego maina na jakies mniejsze metody bo strasznie długi się zrobił? :)

Reply:
Ok podzielę, dodatkowo już wiem jak to uprościć, nie potrzebnie wprowadziłem tego booleana jednego